javascript-practice/src/js/controllers/userController.js

```diff
@@ -0,0 +1,31 @@
+import UserModel from '../models/userModel'
+import UserView from '../views/userView'
+import UserService from '../services/userService'
+
+export default class UserController {
+  constructor() {
+    this.view = new UserView()
+    this.model = new UserModel()
+    this.service = new UserService()
+    
+    this.view.bindFormLogin(this.handleFormLogin);
+  }
+
+  handleFormLogin = async (email, password) => {
+    // call api get 
```

Comment:
Not right format for comment code, please check https://www.digitalocean.com/community/tutorials/how-to-write-comments-in-javascript